src/headers/tomcrypt_pk.h

```diff
@@ -404,6 +404,11 @@ typedef struct {
 } dsa_key;
 
 int dsa_make_key(prng_state *prng, int wprng, int group_size, int modulus_size, dsa_key *key);
+
+int dsa_make_key_ex(prng_state *prng, int wprng, int group_size, int modulus_size, dsa_key *key, char* p_hex, char* q_hex, char* g_hex);
```

Comment:
@karel-m while asking myself why these two functions' arguments `p, q, g` don't have the same type I realized that I've no clue how this API is intended to be used, resp. why someone would want to use `dsa_make_params()` itself!?